src/devices/bus/coco/cococart.cpp

```diff
@@ -443,6 +445,21 @@ device_cococart_interface::~device_cococart_interface()
 
 
 //-------------------------------------------------
+//  interface_config_complete
+//-------------------------------------------------
+
+void device_cococart_interface::interface_config_complete()
+{
+	m_owning_slot = dynamic_cast<cococart_slot_device *>(device().owner());
+	if (!m_owning_slot)
+		throw new emu_fatalerror("Expected device().owner() to be of type cococart_slot_device");
+	m_host = dynamic_cast<device_cococart_host_interface *>(m_owning_slot->owner());
```

Comment:
You'll need to check that `m_owning_slot` isn't null before doing this because you might be instantiated in an otherwise empty machine by an auxiliary verb.